manager/integration/tests/conftest.py

```diff
@@ -174,6 +179,15 @@ def pytest_collection_modifyitems(config, items):
             if "upgrade" in item.keywords:
                 item.add_marker(skip_upgrade)
 
+    if not config.getoption(INCLUDE_CA_OPT):
+        skip_upgrade = pytest.mark.skip(reason="include " +
```

Comment:
Should use right var name for each skipped test cases instead of `skip_upgrade`. Please fix this in `pytest_collection_modifyitems` generally later.